### DISCUSSION TITLE
deposit-cycles should accept TC shorthand for <CYCLES> argument

### DISCUSSION THREAD

**Discussion by DanielThurau** [dfinity/sdk#2791] [Feature Requests]
Writing how many cycles you want to send is cumbersome if you want just want to send a whole number. Being able to specify a whole number with TC shorthand would reduce user input error.

Current experience:
```
dfx canister --network ic deposit-cycles 1000000000000 <CANISTER>
```

Suggested equivalent command:
```
dfx canister --network ic deposit-cycles 1TC <CANISTER>
```

**Comment by ghost** [comment]
Implemented in https://github.com/dfinity/sdk/pull/3452
